go_test.go

```diff
@@ -0,0 +1,171 @@
+package semdiffstat
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestGo(t *testing.T) {
+	tests := []struct {
+		fileA   []byte
+		fileB   []byte
+		changes []*Change
+		err     error
+	}{
+		{ // 1
+			[]byte(fileA),
+			[]byte(fileB),
+			[]*Change{
+				&Change{Name: "func aaaa", InsLines: 3, DelLines: 1, Inserted: false, Deleted: false, IsOther: false},
```

Comment:
You don't need to spell out the zero values here (e.g. `false`), which I think would make this easier to read.